docs/ReleaseNotes.md

```diff
@@ -0,0 +1,30 @@
+# Release notes
+
+## v0.9 
+Excited to announce the release of admiral version `v0.9` with lots of cool functionality added and it is ready for production usage. Following are the highlights of this release:
```

Comment:
We are excited to announce the release of admiral version `v0.9` with lots of cool functionality added. This version is ready for production usage and addresses some of the biggest requests from our users. The following are the highlights of this release: